lib/Jsrt/JsrtCommonExports.inc

```diff
@@ -115,5 +115,4 @@
     JsRunSerialized
     JsCreatePropertyId
     JsCopyPropertyId
```

Comment:
Windows static lib support is couple of changes away. Once its there, JsDiagEvaluate might be better defined with the others. Good point anyways, Thanks!